search.client.ts

```diff
@@ -454,7 +454,7 @@ class OramaSearch {
     );
     return text.replace(
       regex,
-      '<mark class="bg-runtime px-1 py-0.5 rounded text-black font-bold">$1</mark>',
```

Comment:
Less padding on the sides prevents partial-word highlights from looking goofy (e.g, if the user enters "Types" and gets a hit for "TypeScript")